<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>text_ft_main</class>
 <widget class="QWidget" name="text_ft_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>text_ft_main</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Qt Dark Theme Style Sheet */

QWidget {
    background-color: #2b2b2b;
    color: #ffffff;
    font-size: 14px;
    font-family: &quot;Segoe UI&quot;, Arial, sans-serif;
}

QFrame {
    background-color: #3a3a3a;
    border: 1px solid #555;
    border-radius: 8px;
    padding: 6px;
}

QPushButton {
    background-color: #3a3a3a;
    border: 1px solid #555;
    border-radius: 6px;
    padding: 6px 12px;
    color: white;
}

QPushButton:hover {
    background-color: #505050;
    border: 1px solid #777;
}

QPushButton:pressed {
    background-color: #666;
    border: 1px solid #888;
}

QLineEdit {
    background-color: #3a3a3a;
    border: 1px solid #555;
    border-radius: 4px;
    padding: 4px;
    color: white;
}

QLineEdit:focus {
    border: 1px solid #777;
}

QLabel {
    font-size: 14px;
}

QComboBox {
    background-color: #3a3a3a;
    border: 1px solid #555;
    border-radius: 4px;
    padding: 4px;
    color: white;
}

QComboBox:hover {
    border: 1px solid #777;
}

QComboBox QAbstractItemView {
    background-color: #3a3a3a;
    border: 1px solid #555;
    selection-background-color: #505050;
    color: white;
}

QScrollBar:vertical {
    border: none;
    background: #2b2b2b;
    width: 10px;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #505050;
    min-height: 20px;
    border-radius: 5px;
}

QScrollBar::handle:vertical:hover {
    background: #666;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: none;
    border: none;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QFrame" name="text_holder">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTextEdit" name="main_text"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="horizontalWidget_4" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>210</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>210</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="formatter_holder">
           <property name="minimumSize">
            <size>
             <width>762</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>200</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="sentenceCase">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>Sentence case</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="lowerCase">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>lower case</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="upperCase">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>UPPER CASE</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="capitalizedCase">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>Capitalized Case</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="altCase">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>aLtErNaTiNg cAsE</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="titleCase">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>Title Case</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="inverseCase">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>InVeRsE CaSe</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="reverseCase">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>Reverse Case</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="downloadBtn">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>Download</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
